pyop2/global_kernel.py

```diff
@@ -331,14 +333,68 @@ def code_to_compile(self):
         from pyop2.codegen.rep2loopy import generate
 
         wrapper = generate(self.builder)
+        if self._iterset._extruded:
```

Comment:
I think just `self._extruded`

Reply:
Do we want to change that in https://github.com/OP2/PyOP2/blob/944c6cf73b302a1c6b2a5a4bc919db6818e8f96c/pyop2/global_kernel.py#L437 too?